queries/roc/textobjects.scm

```diff
@@ -0,0 +1,35 @@
+(anon_fun_expr
+  (expr_body) @function.inner
+) @function.outer
+
+(argument_patterns
+  ((_) @parameter.inner . ","? @parameter.outer) @parameter.outer
+)
+
+(function_type
+  ((_) @parameter.inner . ","? @parameter.outer) @parameter.outer(#not-eq? @parameter.inner "->")
+)
+
+(function_call_expr
+  .
+  (_)
+  (parenthesized_expr (expr_body) @parameter.inner) @parameter.outer
+)
+
+(function_call_expr
+  .
+  (_) ((_) @parameter.inner) @parameter.outer
+)
+
+[
+  (annotation_type_def ) @class.inner
+  (alias_type_def ) @class.inner
+  (opaque_type_def ) @class.inner
+] @class.outer
```

Comment:
I think this query is a bit misleading, it basically just applies `@class.inner` and `@class.outer` to all elements in this list. Would you mind either moving it to be `[ ... ] @class.inner @class.outer` or doing something else you feel appropriate if you meant there to be a difference between these queries? I don't know this language at all so do what you think is best